boot/build.go

```diff
@@ -131,8 +131,12 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		result.Layers = append(result.Layers, classpathLayer)
 
 	} else {
+		scbJar := FindSpringCloudBindingsJars(d)
+		if scbJar != nil {
+			b.Logger.Headerf("A Spring Cloud Bindings library was found in the Spring Boot libs with this version: %s - not adding another one", scbJar.Version)
```

Comment:
Do you think it important that the version is given in the output here? I'm thinking that it might be enough to let the user know the buildpack found the lib and that it will not contribute one.

Reply:
no, not really. Let's keep things simple and focused